app/src/test/java/org/mozilla/fenix/tabstray/FloatingActionButtonBindingTest.kt

```diff
@@ -134,6 +134,6 @@ class FloatingActionButtonBindingTest {
         verify(exactly = 0) { actionButton.hide() }
         verify(exactly = 1) { actionButton.setText(R.string.sync_syncing_in_progress) }
         verify(exactly = 2) { actionButton.setIconResource(R.drawable.ic_fab_sync) }
-        verify(exactly = 2) { actionButton.contentDescription = any() }
+        verify(exactly = 4) { actionButton.contentDescription = any() }
```

Comment:
Why are there 2 new sets of the content description now?

Reply:
Good question. I didn't really investigate this when refactoring, but I'll check if it can just be the one check.

Reply:
Please do. It was there at one point, and it might have been removed.

Reply:
Update: I misunderstood the question originally. The content description wasn't being set for the "Sync" and "Syncing" states of the FAB previously, so with me setting that now in FloatingActionButtonBinding line 73, it now gets called when the TabsTray goes to synced tabs but also when the TabsTrayStore.syncing variable is updated.

Reply:
Ah ha, great catch! Surprising that it isn't 3 though.